specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -456,15 +489,22 @@ namespace Microsoft.Windows.PushNotifications
         // Register an activator using an ActivationInfo context and return a RegistrationToken
         static PushNotificationRegistrationToken RegisterActivator(PushNotificationActivationInfo details);
 
-        // Unregister any activator if present using a token and registrationKind
+        // Unregister any activator if present using a token and PushNotificationRegistrationOptions
         // 1) If kind = PushTrigger is specified, the trigger itself will be removed
-        // 2) If kind = ComActivator is specified, the Windows App SDK Background Task component will no longer act as an InProc COM Server
-        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationKind kind);
+        // 2) If kind = ComActivator is specified, the Project Reunion Background Task component will no longer act as an InProc COM Server
+        // 3) If kind = ProtocolActivator is specified, the application will be unregistered from the long running process that handles activation
```

Comment:
@pmpurifoy should we also explicitly mention token to be nullptr for ProtocolActivator?